tests/functional/builtins/codegen/test_keccak256.py

```diff
@@ -1,6 +1,10 @@
 from vyper.utils import hex_to_int
 
 
+def test_sanity(keccak):
+    assert keccak(b"").hex() == "c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
```

Comment:
add comment - sha3 vs keccak
```suggestion
    # ensure keccak is keccak256, not sha3
    # https://ethereum.stackexchange.com/a/107985
    assert keccak(b"").hex() == "c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
```